File: searching/linear-search.py
my_list = [4,8,3,7,1,0,9]

ele = int(input())

flag = False
for i in my_list:
    if(i == ele):
        flag = True
        break

if(flag==True):
    print("Element found")
else:
    print("Element not found")

File: searching/linear_search.cpp
#include <iostream>
using namespace std;
int main()
{
    int arr[] = {3, 2, 1, 10, 9};
    bool flag = false;
    for (int i = 0; i < 5; i++)
    {
        if (arr[i] == 2)
        {
            cout << "Found";
            flag = true;
        }
    }
    if (!flag)
    {
        cout << "Not Found";
    }

    return 0;
}